src/disemvowling.js

```diff
@@ -1,38 +1,8 @@
 'use strict'
 
-// export const Disemvoweling {
-//   remove() {
-//     return str.replace(/[aeiou]/gi, "")
-//   }
-// }
-
-// export default class Disemvoweling {
-//   constructor() {
-//     this.str = str
-//   }
-
-//   remove() {
-//     return str.replace(/[aeiou]/gi, "")
-//   }
-// }
-
-// class Disemvoweling {
-//   constructor() {
-//     this.str = str
-//   }
-
-//   remove() {
-//     str = "Hello World"
-//     return str.replace(/[aeiou]/gi, "")
-//   }
-// }
-
-// export default Disemvoweling
-
 export default function Disemvoweling(str) {
   let removeSpace = str.replace(/\s/g, '')
   let removeVowel = removeSpace.replace(/[aeiou]/gi, '')
   return removeVowel
```

Comment:
What about capital vowels? Based on this implementation they would not be removed. You could use the `toLowerCase()` method after the `removeSpace` method. (e.g. `let lowerString = removeSpace.toLowerCase()` and then... `let removeVowel = lowerString.replace(/[aeiou]/gi,'')`)